lib/client.js

```diff
@@ -210,6 +210,136 @@ class Client {
     const relativePathWithQuery = `${params.resourceType}?${new URLSearchParams(params.searchParams)}`;
     return this.httpClient.get(relativePathWithQuery);
   }
+
+  /**
+   * Create a resource.
+   *
+   * @example
+   * const newPatient = {
+   *   resourceType: 'Patient',
+   *   active: true,
+   *   name: [{ use: 'official', family: ['Coleman'], given: ['Lisa', 'P.'] }],
+   *   gender: 'female',
+   *   birthDate: '1948-04-14',
+   * }
+   *
+   * // Using promises
+   * fhirClient.create({
+   *   resource: newPatient,
+   * }).then((data) => { console.log(data); });
+   *
+   * // Using async
+   * let response = await fhirClient.create({
+   *   resource: newPatient,
+   * })
+   * console.log(response);
+   *
+   * @param {Object} params - The request parameters.
+   * @param {String} params.resource - The new resource data to create.
+   *
+   * @return {Promise<Object>} FHIR resource
+   */
+  create(params) {
+    return this.httpClient.post(params.resource.resourceType, params.resource);
+  }
```

Comment:
Should the signature be `create(resource)` -- with no intervening `params` ?

(See @bkaney comment: https://github.com/Vermonster/node-fhir-client/pull/27#issuecomment-379364516)